keyboards/sneakbox/lilbae/keyboard.json

```
@@ -0,0 +1,66 @@
+{
+    "manufacturer": "Sneakbox",
+    "maintainer": "mujimanic",
+    "bootloader": "atmel-dfu",
+    "build": {
+        "lto": true
+    },
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "D2", "pin_b": "D3"}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
```

`nkro` doesnt add much value when there are less than 6 keys
```suggestion
        "mousekey": true
```